### PR TITLE
Rundall/svc 20919/ad hoc sysctl settings

### PR DESCRIPTION
**Pull request by jakerundall**
Tested on hli-cn01 via holli-control. See example data in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,17 @@ include ::profile_hardening
...
+profile_hardening::sysctl::settings:
+  "kernel.perf_event_paranoid":
+    comment: "reference a ticket, CVE, etc."
+    ensure: "present"
+    target: "/etc/sysctl.d/96-kernel-perf_event_paranoid.conf"
+    value: 3
+```
 
 
 ## Dependencies
```
Comment (billglick) [inline README.md:46]: Do we need to add the sysctl Puppet module here?

File: README.md [hunk 2]
```diff
@@ -30,7 +30,17 @@ include ::profile_hardening
 
 ## Usage
 
-The goal is that no paramters are required to be set. The default paramters should work for most NCSA deployments out of the box.
+The goal is that no parameters are required to be set. The default parameters should work for most NCSA deployments out of the box.
+
+To set temporary or Ad-hoc sysctl settings related to security, you can follow this Hiera example:
+```yaml
+profile_hardening::sysctl::settings:
+  "kernel.perf_event_paranoid":
```
Comment (billglick) [inline README.md:38]: I'd make this a bit more generic

**Review by billglick — CHANGES_REQUESTED** [review]
A couple of very minor suggested changes.

Once those are made, feel free to merge it.

File: data/common.yaml [hunk 3]
```diff
@@ -5,3 +5,5 @@ profile_hardening::remove_setuid_setgid::files:
   - "/usr/bin/nvidia-modprobe"
   - "/usr/libexec/Xorg.wrap"
   - "/usr/sbin/login_duo"
+
+profile_hardening::sysctl::settings: []
```
Comment (bsper2) [inline data/common.yaml:9]: Nitpicky but could you set it to this to show it's a hash?
```suggestion
profile_hardening::sysctl::settings: {}
```

**Review by bsper2 — APPROVED** [review]
Pre-approving since it looks good.
Made one small suggestion.

File: README.md (hunk 2 above)
Comment (jakerundall) [reply README.md]: Done

File: data/common.yaml (hunk 3 above)
Comment (jakerundall) [reply data/common.yaml:9]: Good catch! I missed that, but I've opened a new pull request: https://github.com/ncsa/puppet-profile_hardening/pull/3

File: README.md (hunk 1 above)
Comment (jakerundall) [reply README.md:46]: Done! I did include in metadata.json, but good to include in the README.md as well.

**Pull request merged**